topics/single-cell/tutorials/scrna-case_JUPYTER-trajectories/tutorial.md

```diff
@@ -43,190 +43,251 @@ contributions:
     - mtekman
   editing:
     - hexylena
-
-  testing:
     - wee-snufkin
```

Comment:
this looks a lot more like authorship than editing to me, given that you've rewritten half the tutorial and added large new sections :)

Reply:
AGREED!